src/Writers/HTMLWriter.jl

```diff
@@ -584,6 +588,8 @@ function domify_doc(ctx, navnode, md::Markdown.MD)
             ret = Any[domify(ctx, navnode, Writers.MarkdownWriter.dropheaders(markdown))]
             # When a source link is available then print the link.
             Utilities.unwrap(Utilities.url(ctx.doc.internal.remote, ctx.doc.user.repo, result)) do url
+                # Replace any backslashes in links, if building the docs on Windows
+                url = replace(url, '\\', '/')
```

Comment:
Couldn't we move this into the 5-argument `url` function (related to the previous comment)?